pulseapi/entries/serializers.py

```diff
@@ -61,6 +61,14 @@ class EntrySerializer(serializers.ModelSerializer):
                                          queryset=Creator.objects,
                                          required=False)
 
+    published_by = serializers.SerializerMethodField()
```

Comment:
This is fine, although it's a good idea to say here that we're "rebinding" or "overriding" what `published_by` does for REST purposes, compared to what that field does when working directly with a model instance.